hotel-management/src/pages/HomePage/index.tsx

```diff
@@ -1,41 +1,17 @@
 import { Link } from 'react-router-dom';
```

Comment:
Have u tested UI when loading or error yet, and any image attach for this UI render?

Reply:
Yes I did, and the fix will be on another pr